content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisstack/redisstack-7.4-release-notes.md

```diff
@@ -0,0 +1,131 @@
+
+[Docker](https://hub.docker.com/r/redis/redis-stack) | [Download](https://redis.io/downloads/#stack)
+
+### Headlines
+
+This version includes security fixes for the **Redis** server, addressing potential vulnerabilities such as an RCE when using Lua library components, and a denial-of-service (DoS) risk due to malformed ACL selectors or unbounded pattern matching.
+Additionally, this maintenance release includes the latest version of **Redis Insight**.
+
+### Details
+
+**Security and privacy**
+* **Redis**:
+  * (CVE-2024-31449) Lua library commands may lead to stack overflow and potential RCE.
+  * (CVE-2024-31227) Potential Denial-of-service due to malformed ACL selectors.
+  * (CVE-2024-31228) Potential Denial-of-service due to unbounded pattern matching.
+	
+**Redis Community Edition version**
+  * [Redis 7.4.1](https://github.com/redis/redis/releases/tag/7.4.1)
+
+**Module versions**	
+* [RediSearch 2.10.5](https://github.com/RediSearch/RediSearch/releases/tag/v2.10.5)
+* [RedisJSON 2.8.3](https://github.com/RedisJSON/RedisJSON/releases/tag/v2.8.3)
+* [RedisTimeSeries 1.12.2](https://github.com/RedisTimeSeries/RedisTimeSeries/releases/tag/v1.12.2)
+* [RedisBloom 2.8.2](https://github.com/RedisBloom/RedisBloom/releases/tag/v2.8.2)
+
+**Recommended client libraries**
+* Java
+  * [Jedis 5.2.0 or later](https://github.com/redis/jedis/releases/tag/v5.2.0)
+  * [Lettuce 6.4.0 or later](https://github.com/redis/lettuce/releases/tag/6.4.0.RELEASE)
+* Python
+  * [redis-py 5.1.0 or later](https://github.com/redis/redis-py/releases/tag/v5.1.0)
+* NodeJS
+  * [node-redis 4.7.0 or later](https://github.com/redis/node-redis/releases/tag/redis%404.7.0)
+* Go
+  * [go-redis 9.6.1 or later](https://github.com/redis/go-redis/releases/tag/v9.6.1)
+
+Compatible with [Redis Insight](https://redis.io/download). The docker image redis/redis-stack for this version is bundled with [Redis Insight 2.58](https://github.com/RedisInsight/RedisInsight/releases/tag/2.58.0).
+
+Note: version numbers follow the pattern:
+
+`x.y.z-b`
+* `x.y` Redis Major version
+* `z` increases with even numbers as a module x.y version increases.
+* `b` denotes a patch to Redis or a module (any `z` of Redis or modules). `b` will consist of a `v` + numeric value.
+
+## Redis Community Edition 7.4.0-v0 (July 2024)
+
+This is a GA release of Redis Stack version 7.4.
+
+[Docker](https://hub.docker.com/r/redis/redis-stack) | [Download](https://redis.io/downloads/#stack)
+
+### Headlines
+**Data Structures**
+
+`Hash`: Redis now supports expiration of individual hash fields. Redis already supports key expiration. For each key - users can specify a time when the key should expire, or alternatively - specify the remaining time-to-live (TTL) after which the key would expire. One very frequent request was to allow specifying expiration time or TTL also for individual hash fields, which is now supported using 9 new Redis commands:
```

Comment:
Are the dashes intended to make a bullet list? It looks a bit odd with them dividing up the sentence like that.

Also, is the "9" digit acceptable in rel notes?

Reply:
Fixed in next merge.